src/sentry/tasks/weekly_reports.py

```diff
@@ -913,6 +918,17 @@ def sum_event_counts(project_ctxs):
                     }
                 )
             series.append((to_datetime(t), project_series))
+        if ctx.organization.slug == "sentry":
+            # TODO(isabella): remove debug logging for sentry
+            logger.info(
+                "render_template_context.trends.totals",
+                extra={
+                    "error_count": total_error,
+                    "transaction_count": total_transaction,
+                    "replay_count": total_replays,
+                    "project_count": len(projects_associated_with_user),
+                },
```

Comment:
are you going to look this up by org slug? might need that in the extra

Reply:
atm i only have it log for the sentry org; gonna send a weekly report email to myself, and then in the situation that i can't get any new information from that, change the logging here to be for all orgs (and add the slug in the extra).

i'd just rather not log for every single org if i don't need to

Reply:
ah okay just making sure you can find it later 👍